docs/tutorial.rst

```diff
@@ -1,16 +1,67 @@
 Start Here: ``datascience`` Tutorial
-======
+====================================
 
-In progress.
+This is general introduction to the functionality in
+:py:mod:`datascience`.  For a reference guide, please see
+:ref:`tables-overview`.
 
 Introduction
-------
+------------
+
+First, load the :py:mod:`datascience` module:
+
+.. ipython:: python
+
+   import datascience as ds
+
+In the IPython notebook, type `ds.` followed by the TAB-key to see a list of members.
+The most important of these is the :py:class:`Table` class, which is
+the structure used to represent columns of data.
```

Comment:
We've always used `from datascience import *` with the students so they've never seen the `ds.Table` syntax before. We'll definitely want to avoid dropping an entirely new syntax on students since we're in the middle of the semester. I'm fine with doing something like this next semester if @papajohn is cool with it.